docs/changelog/116591.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 116591
+summary: "[ES|QL] Add support for `byte_length` scalar function"
```

Comment:
Also, the convention for functions is capitalized `BYTE_LENGTH`.

Reply:
Re the `[ES|QL]` prefix, it took this automatically from the git commit. I can change it, but for the commit I do think it's useful, since when doing something like `git log`, it's useful to have that prefix for huge projects like this one.

Reply:
This text goes into the changelog. And there it is grouped into groups by the `area` line. When we used to manage releases ourselves, I remember putting a lot of effort into deleting all these `ES|QL` prefixes from the yaml files. Obviously we do not change the PR titles, and the commit log messages can remain more explicit. It is only the changelog entry that I think should change.

Reply:
The git commit for the final git log is created at `squash and merge` time, so there is the opportunity to make the best commit message. Thaty should be quite independant of the changelog message here.